benchmarks/differenciation/melt_srb.cc

```diff
@@ -0,0 +1,698 @@
+/*
+  Copyright (C) 2015 - 2018 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/material_model/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/postprocess/melt_statistics.h>
+#include <aspect/melt.h>
+
+
+
+
+#include <aspect/material_model/melt_global.h>
+#include <aspect/adiabatic_conditions/interface.h>
+
+#include <deal.II/base/parameter_handler.h>
+#include <deal.II/numerics/fe_field_function.h>
+
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+    /**
+     * A material model that implements a simple formulation of the
```

Comment:
This is likely copy-pasted from the melt global model, please replace by a short sentence saying this is a modification of melt global for the benchmark you are running. Also leave a sentence what you changed from melt global.